### PR TITLE
Back button

### PR DESCRIPTION
**Pull request by Wixx19**
- background added
- video playback delayed by 2 seconds
- button ready to go on into the next scene
- commented out code that would be used for a potential new button

### CODE REVIEW

**Comment by mariana-perezh** [comment]
I merged this to the game scene that I'm working on in branch mission1_game. Not sure if we should merge right now or wait until it is done.

**Comment by Wixx19** [comment]
We should merge it unless it becomes difficult to make changes later on. We got most of the requirements done.

**Comment by mariana-perezh** [comment]
I'll approve in a few hours

File: Missions/mission_1/task1.gd [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+	start_video_with_delay()
 
 
 # Called every frame. 'delta' is the elapsed time since the previous frame.
 func _process(delta):
 	pass
+
+
+func _on_button_pressed():
+	get_tree().change_scene_to_file(""res://Missions/mission_1/mission_1_game/game.tscn"")
```
Comment (mariana-perezh) [inline Missions/mission_1/task1.gd:19]: Two ""

File: Missions/mission_1/Button.gd
Comment (mariana-perezh) [inline Missions/mission_1/Button.gd:1]: Is this unnecessary?

**Review by mariana-perezh — CHANGES_REQUESTED** [review]

File: Missions/mission_1/Button.gd
Comment (Wixx19) [reply Missions/mission_1/Button.gd:1]: isn't not the button that takes the user back to the map?

Comment (mariana-perezh) [reply Missions/mission_1/Button.gd:1]: this is an extra script i think but let me do something real quick

**Review by mariana-perezh — APPROVED** [review]

**Comment by mariana-perezh** [comment]
Try it on your machine before you merge but it should be good

**Comment by Wixx19** [comment]
It's all good.

**Pull request merged**